pkg/ddc/alluxio/operations/base.go

```diff
@@ -327,6 +328,32 @@ func (a AlluxioFileUtils) Count(alluxioPath string) (fileCount int64, folderCoun
 	return int64(ufileCount), int64(ufolderCount), int64(utotal), err
 }
 
+// file count of the Alluxio Filesystem (except folder)
+// use "alluxio fsadmin report metrics" for better performance
+func (a AlluxioFileUtils) GetFileCount() (fileCount int64, err error) {
+	var (
+		command = []string{
+			"alluxio", "fsadmin", "report", "metrics", "|", "grep", "Master.TotalPaths"}
+		stdout string
+		stderr string
+	)
+
+	stdout, stderr, err = a.execWithoutTimeout(command, false)
+	if err != nil {
+		err = fmt.Errorf("execute command %v with expectedErr: %v stdout %s and stderr %s", command, err, stdout, stderr)
+		return
+	}
+
+	// eg: Master.FilesCompleted  (Type: COUNTER, Value: 6)
+	matchExp := regexp.MustCompile(`\d+`)
+	fileCountStr := matchExp.FindString(stdout)
+	fileCount, err = strconv.ParseInt(fileCountStr, 10, 64)
```

Comment:
Related alluxio metric looks like this when its value is large:
`Master.FilesCompleted  (Type: COUNTER, Value: 3,804,847) `
So the regexp here can't extract the actual number of the metric.

Reply:
Thanks for your reminding.Maybe using "str.replace(',', '')" first can slove this problem.